driver/src/contracts/stablex_contract.rs

```diff
@@ -288,13 +290,15 @@ impl StableXContract for StableXContractImpl {
         Ok(())
     }
 
-    fn past_events(
+    fn events(
```

Comment:
nit: I'd still call it getPastEvents since the toBlock can not really be in the future, can it?